contents/handbook/engineering/support-hero.md

```diff
@@ -4,35 +4,59 @@ sidebar: Handbook
+As Support Hero, your job is to investigate and resolve issues reported by customers. A single case of suspicious data or a show-stopping bug can really undermine one's confidence in a data product, so it's important that we get to the bottom of all issues.
 
-[Marcus](https://posthog.com/community/profiles/1036), our Support Engineer, will triage tickets for the Product Analytics and Pipeline team, due to the high volume of tickets those teams get. He will resolve tickets if possible, and escalate to the Engineering team responsible if he needs further help.
+You'll see some teams using a term of endearment for Support Hero, examples being "Infra Hero" or… "Luigi". Don't ask – we don't know.
 
-### Expectations
+[Marcus](https://posthog.com/community/profiles/1036), our Support Engineer, triages tickets for the Product Analytics and Pipeline team, due to the high volume of tickets those teams get. He will resolve tickets if possible, and escalate to the engineering team responsible if he needs further help.
 
-You should work your 'normal' hours during this week. There will likely be more issues than you'll have time to fix so be sure to prioritise. If there's an important and urgent issue near the end of your day, hand it off to someone on the other side of the Atlantic.
+## When is my turn?
 
-If you are planning on taking a day off or you won't be available, please find someone to swap with and update the rotation on PagerDuty. Be sure to schedule an override for both swaps and **do not alter the rotation order** to avoid affecting other members.
+Most engineering teams run a PagerDuty schedule:
 
-### Rotations
+- [Product Analytics](https://posthog.pagerduty.com/schedules#PXUZ9XL)
+- [Feature Success](https://posthog.pagerduty.com/schedules#P04FUTJ)
+- [Replay](https://posthog.pagerduty.com/schedules#PUPERAV)
+- [Pipeline](https://posthog.pagerduty.com/schedules#PM8YSH8)
+- [Infrastructure](https://posthog.pagerduty.com/schedules#P78OOWZ)
+- [Growth](https://posthog.pagerduty.com/schedules#PN1Q6BO)
 
-- [Secondary - Product Analytics](https://posthog.pagerduty.com/schedules#PXUZ9XL)
-- [Secondary - Feature Success](https://posthog.pagerduty.com/schedules#P04FUTJ)
-- [Secondary - Monitoring](https://posthog.pagerduty.com/schedules#PUPERAV)
-- [Secondary – Pipeline](https://posthog.pagerduty.com/schedules#PM8YSH8)
-- [Secondary - Infrastructure](https://posthog.pagerduty.com/schedules#P78OOWZ)
+The schedules consist of contiguous blocks, but that definitely doesn't mean working 24/7 – you should just work your normal hours.
```

Comment:
Can we write down how Saturdays and Sundays should be since its not 
"normal hours" but part of the schedule anyway.

Reply:
Makes sense to clarify!
```suggestion
The schedules consist of contiguous blocks, but you should just work your normal hours. Support Hero is Monday to Friday only.

Reply:
Thanks, I am part of the `All team escalation` schedule, also for the weekend, I believe All team escalation != Support Hero schedule then?

Reply:
Yes, that's on-call – they're very much separate processes. I renamed "All team escalation" so that this is clearer:
<img width="759" alt="Screenshot 2024-01-04 at 15 22 04" src="https://github.com/PostHog/posthog.com/assets/4550621/9f21d4c1-4440-4534-b943-87490af6a17e">
As for on-call, there's a separate PR cleaning it up, hopefully making it clearer: #7432